lib/web_ui/dev/test_runner.dart

```diff
@@ -240,12 +243,38 @@ class TestCommand extends Command<bool> with ArgUtils {
     }
 
     if (htmlTargets.isNotEmpty) {
-      await _buildTests(targets: htmlTargets, forCanvasKit: false);
+      await _buildTestsInParallel(targets: htmlTargets, forCanvasKit: false);
     }
 
+    // Currently iOS Safari tests are running on simulator, which does not
+    // support canvaskit backend.
     if (canvasKitTargets.isNotEmpty) {
-      await _buildTests(targets: canvasKitTargets, forCanvasKit: true);
+      await _buildTestsInParallel(
+          targets: canvasKitTargets, forCanvasKit: true);
     }
+
+    // Copy image files from test/ to build/test/.
+    // A side effect is this file copies all the images even when only one
+    // target test is asked to run.
+    final List<io.FileSystemEntity> contents =
+        environment.webUiTestDir.listSync(recursive: true);
+    contents.whereType<io.File>().forEach((final io.File entity) {
+      final String basename = path.basename(entity.path);
+      if (basename.contains('.png')) {
```

Comment:
Why not copy all files? I think `.dart` files are also necessary for sourcemaps to work.

Reply:
done, thansk!